### ISSUE TITLE
Is the project active?

### ISSUE THREAD

**Issue by justSteve** [digitalyacht/ikommunicate#7]
I've started off visiting the forum at http://ikommunicate.com/forums - no way to login and no recent posts. From outward appearances not much going on here either. Have just taken delivery of the ikommunicate hardware and wondering if it's orphaned on delivery?

**Comment by sumps** [comment]
Hi Steve,

Sorry for the delay in reply. Please could you contact me directly at paul@digitalyacht.co.uk<mailto:paul@digitalyacht.co.uk>

Best regards

PAUL

**Comment by jfexyz** [comment]
Yes, is this project dead? Thinking about buying an iKommunicate, but not if it’s not being actively developed...

**Comment by sumps** [comment]
iKommunicate is a stable gateway design that was developed to support V1 of Signal K and is not really suitable for developers that want to be at the "cutting edge" of Signal K development. 

For that I would recommend our iKonvert USB gateway which has been natively supported by the Signal K Server since V1.10.0....

[https://digitalyacht.net/2018/12/17/signal-k-node-server-v1-10-0-now-supports-ikonvert-gateway/](https://digitalyacht.net/2018/12/17/signal-k-node-server-v1-10-0-now-supports-ikonvert-gateway/)

With iKonvert you can have full bi-directional NMEA2000 communication between your NMEA2000 network and the Signal K server and is the combination that most Signal K developers are now using.

**Comment by CaptIgmu** [comment]
Ooops!  When I bought my iKommunicate a while ago, we were assured that with further updates it would become "bi-directional." It has not! I am not wanting to be "cutting edge" for Signal K development, I just wanted to put processed data back on to the NMEA2000/0183 bus! It is a gateway . . . remember?

**Comment by kimburgess** [comment]
Similar to the above commenter - I purchased a device a while back as it’s an excellent project and seemed like a good way to fund continued development. Is there a recommended way to repurpose the existing hardware to enable bidirectional comms and continued use with the evolving signal K ecosystem?

**Comment by Max194703** [comment]
Nice rip off. I bought at the time of him enthusiastic iKommunicate. Not being a technician I thought that with the updates (bidirectional ...) I would be able to make the most of it. But I was wrong! I subscribe to what @CaptIgmu says: "I am not wanting to be" cutting edge "for Signal K development, I just wanted to put processed data back on to the NMEA2000 / 0183 bus! It is a gateway... Remember?"
